src/one_wire.toit

```diff
@@ -31,13 +31,22 @@ class Bus:
   */
   static SKIP_FAMILY ::= Object
 
-  protocol_/Protocol? := ?
+  /**
+  The protocol.
+  Use the $protocol_ accessor to use the protocol.
+  */
+  protocol__/Protocol? := ?
+
+  protocol_ --power/bool [block] -> none:
+    if not protocol__: throw "BUS CLOSED"
+    protocol__.set_power power
+    block.call protocol__
 
   /**
   Constructs a 1-wire bus using the given $protocol.
   */
   constructor.protocol protocol:
```

Comment:
Why is the protocol argument not typed?

Reply:
Initially it was using a `.protocol_` (at which point it got the type from the field).
Added the type.
done.